### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by ajrice6713**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-dx-2421.s3-website-us-east-1.amazonaws.com/

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## For the Committer
 
+⚠️⚠️⚠️⚠️⚠️**DEPRECATED**⚠️⚠️⚠️⚠️⚠️
+
+This repository is deprectaed - please use https://github.com/Bandwidth/api-docs
```
Comment (hamermike) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
This repository is deprecated - please use https://github.com/Bandwidth/api-docs
```

**Review by hamermike — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ajrice6713) [reply .github/PULL_REQUEST_TEMPLATE.md]: gonna see if vs code has a better spell check extension 😂

**Review by ckoegel — APPROVED** [review]

**Review by hamermike — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-dx-2421.s3-website-us-east-1.amazonaws.com/

**Pull request merged**